modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -2,6 +2,169 @@
 ///CODE FOR LEWD QUIRKS///
 //////////////////////////
 
+/////////////////
+///BIMBO TRAIT///
+/////////////////
+
+/datum/brain_trauma/special/bimbo
+	name = "Permanent hormonal disruption"
+	desc = "The patient has completely lost the ability to form speech and seems extremely aroused."
+	scan_desc = "permanent hormonal disruption"
+	gain_text = span_purple("Your thoughts get cloudy, but it turns you on like hell.")
+	lose_text = span_warning("A pleasant coolness spreads throughout your body, You are thinking clearly again.")
+	can_gain = TRUE
+	random_gain = FALSE
+	resilience = TRAUMA_RESILIENCE_LOBOTOMY
+	///how satisfied the person is, gained through climaxing
+	//max is 300, min is 0
+	var/satisfaction = 300
+	///how stressed the person is, gained through zero satisfaction
+	//max is 300, min is 0
+	var/stress = 0
+
+	COOLDOWN_DECLARE(desire_cooldown)
+	///The time between each desire message within company
+	var/desire_cooldown_number = 30 SECONDS
+	///The list of manual emotes that will be done when unsatisfied
+	var/static/list/lust_emotes = list(
+		"pants as their body trembles lightly.",
+		"lightly touches themselves up and down, feeling every inch.",
+		"puts their finger in their mouth and slightly bites down.",
+		"places their hands on their hip as they slowly gyrate.",
+		"moans, their head tilted slightly."
+	)
+
+/**
+ * If we are not satisfied, this will be ran through
+ */
+/datum/brain_trauma/special/bimbo/proc/try_unsatisfied()
+	var/mob/living/carbon/human/human_owner = owner
+	//we definitely need an owner; but if you are satisfied, just return
+	if(satisfaction || !human_owner)
+		return FALSE
+	//we need to feel consequences for being unsatisfied
+	//the message that will be sent to the owner at the end
+	var/lust_message = "Your breath begins to feel warm..."
+	//we are using if statements so that it slowly becomes more and more to the person
+	human_owner.manual_emote(pick(lust_emotes))
+	if(stress >= 60)
```

Comment:
could this be a switch?

Reply:
could, but would result in a lot of repeat code.